internal/services/oracle/cloud_vm_cluster_resource.go

```diff
@@ -237,6 +248,22 @@ func (CloudVmClusterResource) Arguments() map[string]*pluginsdk.Schema {
 			ForceNew: true,
 		},
 
+		"scan_listener_port_tcp": {
+			Type:         pluginsdk.TypeInt,
+			Optional:     true,
+			Computed:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.IntAtLeast(0),
+		},
+
+		"scan_listener_port_tcp_ssl": {
+			Type:         pluginsdk.TypeInt,
+			Optional:     true,
+			Computed:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.IntAtLeast(0),
+		},
```

Comment:
`Computed` should only be used in specific cases. Since these have defined default values we should be setting those here. 
```suggestion
		"scan_listener_port_tcp": {
			Type:         pluginsdk.TypeInt,
			Optional:     true,
			Default:     1521,
			ForceNew:     true,
			ValidateFunc: validation.IntAtLeast(0),
		},

		"scan_listener_port_tcp_ssl": {
			Type:         pluginsdk.TypeInt,
			Optional:     true,
			Default:     2484,
			ForceNew:     true,
			ValidateFunc: validation.IntAtLeast(0),
		},
```

Reply:
updated